srv/modules/runners/validate.py

```diff
@@ -973,6 +992,146 @@ def report(self):
         self.printer.print_result()
 
 
+class ConfigCheck(object):
+    """Class to detect deprecated config values in files.
+
+    Attributes:
+        base_path (str): Base path
+        map_file (str): Map file path
+        conf_path (str): Conf file path
+        suffix (str): Suffix for config files
+        files (list): List of files from glob
+        map (dict): Map of deprecated k:v
+        issues (list): List of found incidents
+    """
+    def __init__(self):
+
+        self.base_path = '/srv/salt/ceph/configuration/files'
+        self.map_file = '{}/deprecated_map.yml'.format(self.base_path)
+        self.conf_path = '{}/ceph.conf.d'.format(self.base_path)
+        self.suffix = '.conf'
+        self.files = glob.glob("{path}/*{suffix}".format(path=self.conf_path,
+                                                         suffix=self.suffix))
+        self.map = self.load_map()
+        self.issues = []
+
+    def load_map(self):
+        """
+        Loads k:v map from disk
+
+        Returns:
+            YAML map
+        Raises:
+            YAMLError
+        """
+        with open(self.map_file, 'r') as _fd:
+            try:
+                return yaml.load(_fd)
+            except yaml.YAMLError as exc:
+                raise exc
```

Comment:
I don't really understand the point of re-raising the YAMLError here. Is it to lose the backtrace by raising a new YAMLError?